.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Don't know the notation, but it should be on all PRs, nope ?
```suggestion
    branches: *
```

---

Good point. That comes from the [Node.js CI starter template](https://github.com/actions/starter-workflows/blob/main/ci/node.js.yml) provided by GitHub. But surprisingly, the CI works fine on that branch.

---

Ok I figured it out 🤯. GitHub Actions evaluate the **base** branch to determine if the workflow must be launched.